__tests__/Str-test.js

```diff
@@ -95,3 +95,11 @@ describe('Str.sanitizeURL', () => {
         expect(Str.sanitizeURL('HTtp://FOO.com/blah_BLAH')).toBe('http://foo.com/blah_BLAH');
     });
 });
+
+describe('Str.isValidEmail', () => {
+    it('Correctly detects a valid email', () => {
+        expect(Str.isValidEmail('abc@gmail.com')).toBeTruthy();
+        expect(Str.isValidEmail('test@gmail')).toBeFalsy();
+        expect(Str.isValidEmail('usernamelongerthan64charactersshouldnotworkaccordingtorfc822whichisusedbyphp@gmail.com')).toBeFalsy();
```

Comment:
NAB: Should we add a test for usernames shorter than 1 character?

Reply:
Ah yes, it makes sense to add that test for completeness. Thanks 🙇 
Follow-up PR [here](https://github.com/Expensify/expensify-common/pull/563)